packages/react-link/src/components/Link/Link.test.tsx

```diff
@@ -19,12 +15,68 @@ describe('Link', () => {
     }
   });
 
-  /**
-   * Note: see more visual regression tests for Link in /apps/vr-tests.
-   */
-  it('renders a default state', () => {
-    const component = renderer.create(<Link>Default Link</Link>);
+  isConformant({
+    Component: Link,
+    displayName: 'Link',
+  });
+
+  describe('AccessibilityLinkBehavior', () => {
+    const testFacade = new ComponentTestFacade(Link, {});
+    const errors = validateBehavior(linkBehaviorDefinition, testFacade);
+    expect(errors).toEqual([]);
+  });
+
+  it('renders as a button if no href is provided', () => {
+    wrapper = mount(<Link>This is a link</Link>);
+    const button = wrapper.find('button');
+    const anchor = wrapper.find('a');
+    expect(button.length).toBe(1);
+    expect(anchor.length).toBe(0);
+
+    const component = renderer.create(<Link>This is a link</Link>);
```

Comment:
Just curious, I've seen this a few times now, why is `react-test-renderer` always used for snapshot testing ? instead of just reusing the wrapper that already exists

Reply:
Whoops yeah, I've removed the duplicate.